<launch>

    <node name="servo_robot" pkg="moveit_servo" type="servo_server_sub_pub" output="screen">
        <!-- <param name="parameter_ns" type="string" value="" /> -->
        <rosparam command="load" file="$(find moveit_servo)/config/ur_simulated_config_frames.yaml"/>
    </node>

    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="test_tf_broadcaster" args="0 0 0.3 0 0 0 1 wrist_3_link TCP_M" /> -->
    <node name="robot_controller_minimal" pkg="moveit_servo" type="robot_controller_minimal.py" output="screen"/>

</launch>